addons/binding/org.openhab.binding.wso2iots/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="wso2iots"
+						  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+						  xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+						  xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="buildingMonitor">
+		<label>Building Monitor Data</label>
+		<description>Provides temperature, motion, humidity, light level collected from building monitoring devices</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="humidity" typeId="humidity"/>
+			<channel id="motion" typeId="motion"/>
+			<channel id="light" typeId="light"/>
+		</channels>
+
+        <!-- meta data -->
+		<properties>
+             <property name="buildingId">buildingId</property>
+             <property name="floorId">floorId</property>
+        </properties>
+        
+		<config-description>
+		    <parameter name="deviceId" type="text" required="true">
+                <label>Device ID</label>
+                <description>ID of the building monitor device</description>
+            </parameter>
+            
+			<parameter name="apikey" type="text" required="true">
+				<context>password</context>
+				<label>API Key</label>
+				<description>Data-platform token to access the IOT server</description>
+			</parameter>
+			
+			<parameter name="refresh" type="integer" min="1" required="false">
+				<label>Refresh interval</label>
+				<description>Specifies the refresh interval in seconds.</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<!-- Sample Channel Type -->
+	
+	<channel-type id="temperature" advanced="true">
+		<item-type>Number</item-type>
+		<label>Temperature (°C)</label>
+		<description>Temperature in Celsius degrees</description>
+		<category>Temperature (°C)</category>
+		<state readOnly="true" pattern="%.1f °C"></state>
+	</channel-type>
+
+	<channel-type id="motion" advanced="true">
+		<item-type>Number</item-type>
+		<label>motion</label>
```

Comment:
Consistency in naming convention: capital or small letters, please unify across this xml for labels, descriptions and categories.